### ISSUE TITLE
Implement colour swatch

### ISSUE THREAD

**Issue by jessjwilliamson** [musescore/MuseScore#17714]
The design is ready to be implemented.
This colour swatch would not be affected by the accent colour - this colour swatch enables users to assign colours to mixer channels, percussion, etc.

Link to design is sent to dev team.

<img width="1135" alt="image" src="https://github.com/musescore/MuseScore/assets/71656189/4e642c90-c38c-4586-8591-91b78bced338">

**Comment by shoogle** [comment]
It would be nice to display the colour name in a tooltip as well as sending it to screen readers.

**Comment by jessjwilliamson** [comment]
> It would be nice to display the colour name in a tooltip as well as sending it to screen readers.

I agree! Update on this task, it is not ready to implement yet (will move this) and there are more questions to solve.

**Comment by bkunda** [comment]
Just a thought about the UX of this (for whoever ends up implementing it).

When we get around to implementing the colour picker in specific instances, I suspect it might be better to include it as a popup rather than a dialog. 

I'm thinking specifically of the new percussion panel here.  

In cases where it _does_ appear as a dialog, I think a "Done" or "Ok" button should be included as an additional mechanism for closing it. 

But we can circumvent this requirement if it takes the form of a popup (where "close" will happen by clicking away from it).
